core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -582,6 +589,80 @@ export class BaseUser {
   }
 
   /**
+   * This function takes a screenshot of the page.
+   * If there's no parameter for newPage, it checks this.page instead of newPage.
+   * If there's no image with the given filename, it stores the screenshot with the given filename in the folder:
+   *  prod_desktop_screenshots or prod_mobile_screenshots for screenshots in production mode
+   *  dev_desktop_screenshots or dev_mobile_screenshots for screenshots in development mode
+   * Otherwise, it compares the screenshot with the image named as the given string to check if they match.
+   * If they don't match, it generates an image in the folder __diff_output__ to show the difference.
```

Comment:
```suggestion
   * If there's no image with the given filename, it stores the screenshot with the given filename in the folder:
   *   - prod_desktop_screenshots or prod_mobile_screenshots for screenshots in production mode
   *   - dev_desktop_screenshots or dev_mobile_screenshots for screenshots in development mode
   * Otherwise, it compares the screenshot with the image named as the given string
   * to check if they match. If they don't match, it generates an image in the folder
   * __diff_output__ to show the difference.
```

Reply:
Done